admin/views/class-yoast-form-fieldset.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Generate the HTML for a form fieldset to wrap grouped form elements.
+ */
+class Yoast_Form_Fieldset implements Yoast_Form_Element {
+
+	/**
+	 * @var string
+	 */
+	private $fieldset_id;
+
+	/**
+	 * @var array The fieldset HTML attributes.
+	 */
+	private $fieldset_attributes = array();
```

Comment:
Given my comment on the `fieldset_add_attribute` function, you should call this `attributes`.